quantus/metrics/axiomatic/completeness.py

```diff
@@ -65,7 +73,7 @@ def __init__(
         normalise_func_kwargs: Optional[Dict[str, Any]] = None,
         output_func: Optional[Callable] = lambda x: x,
         perturb_baseline: str = "black",
-        perturb_func: Callable = None,
+        perturb_func: Callable = baseline_replacement_by_indices,
```

Comment:
using mutable default values for parameters of mutable type is dangerous:
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
i was confused so much the first time I encountered such a bug.

it's actually one of the very few aspects of python i don't like, but i guess there's a weird use case for this...

Reply:
done